### PR TITLE
Update docker images and drone docker build options

### PR DESCRIPTION
**Pull request by jrebey**
Update docker images to use distroless containers and make drone use its own Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
-    rm /tmp/xbvr.tgz
+WORKDIR /app
+ADD . /app
+RUN cd /app && \
+    yarn install && \
+    yarn build && \
+    go generate && \
+    go build -tags='json1' -ldflags '-w' -o xbvr main.go
+
+FROM FROM gcr.io/distroless/base-debian10
```
Comment (cld9x) [inline Dockerfile]: `FROM FROM`? 😄 

Also, why debian10?

Comment (jrebey) [reply Dockerfile]: grrr... I fixed that and then something must have reverted. I'll fix it.

Comment (jrebey) [reply Dockerfile]: Google updated the distroless to debian10 but they pulled the images for some reason so I reverted it back to base.

**Comment by jrebey** [comment]
FYI you can test a container built with this Dockerfile at jrebey/xbvr:latest

**Pull request merged**